src/Compilers/CSharp/Test/Syntax/IncrementalParsing/IncrementalParsingTests.cs

```diff
@@ -2341,6 +2341,51 @@ public void InsertOpenBraceBeforeCodes()
             CompareIncToFullParseErrors(reparsedTree, parsedTree);
         }
 
+        [WorkItem(6676, "https://github.com/dotnet/roslyn/issues/6676")]
+        [Fact]
+        public void InsertExpressionStatementWithoutSemicolonBefore()
+        {
+            SourceText oldText = SourceText.From(@"
+System.Console.WriteLine(true)\n
+");
+            var startTree = SyntaxFactory.ParseSyntaxTree(oldText);
+
+            var newText = oldText.WithChanges(new TextChange(new TextSpan(0, 0), "System.Console.WriteLine(false)\n"));
+            var reparsedTree = startTree.WithChangedText(newText);
+            var parsedTree = SyntaxFactory.ParseSyntaxTree(newText);
```

Comment:
>parsedTree [](start = 16, length = 10)

It would be goof to assert errors from the tree. We have "Verify" helpers to do that.